src/languages/idl.js

```diff
@@ -0,0 +1,58 @@
+/*
+Language: OMG IDL
```

Comment:
Alternatively, we could make sure this covers both IDLs... webIDL is based on OMG IDL, so totally possible.